components/Layout.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+import NavFooter from "./NavFooter";
+
+export default function Layout({ children }) {
+  return (
+    <>
+      <PageWrapper>{children}</PageWrapper>
+      <NavFooter />
+    </>
+  );
+}
+
+const PageWrapper = styled.div`
```

Comment:
Mach aus dem PageWrapper am besten in main Element. Dann kannst du auch das main Element aus deinen ganzen Pages rausnehmen.